documentatie/gebruikersdocumentatie/installeren-en-starten.md

```diff
@@ -0,0 +1,89 @@
+# Abacus installeren en starten
+
+Het installeren en starten van Abacus kan op verschillende manieren, met elk verschillende doeleinden.
+Hieronder vind je instructies voor de verschillende methodes.
+
+## Linux en macOS
+
+De onderstaande methodes werken waarschijnlijk op alle gangbare (recente) Linux-distributies en zijn getest op Ubuntu (22.04 en hoger) en Debian (12/bookworm). Methode 1 werkt niet op macOS omdat we geen macOS-specifieke builds maken, maar de andere methodes werken wel.
+
+## Windows
+
+De methodes 3 en 4 werken ook in Windows. Op dit moment kun je nog geen installatiebestand voor Windows downloaden, maar we zijn hiermee bezig. Zodra deze optie er is vind je hier meer info.
+
+### Methode 1: build artifact downloaden en starten
+
+Deze methode lijkt het meest op het draaien van een productiebuild en is ook het meest eenvoudig. Voer de volgende stappen uit:
+
+1. Ga in de Abacus-repository naar [Actions](https://github.com/kiesraad/abacus/actions).
+2. Klik aan de linkerkant op [Build, lint & test](https://github.com/kiesraad/abacus/actions/workflows/build-lint-test.yml).
+3. Klik op de eerste workflow waarbij de tweede kolom leeg is.
+4. Download het bestand *backend-build*. Let op: de download is alleen zichtbaar als je bent ingelogd in GitHub.
+
+![instructions1](/documentatie/gebruikersdocumentatie/img/build-artifact-1.png)
+![instructions2](/documentatie/gebruikersdocumentatie/img/build-artifact-2.png)
```

Comment:
Waarom hebben de plaatjes Engelstalige labels terwijl we Nederlandstalige documentatie schrijven?

Reply:
Goeie! Dat stond al zo in het document, ik heb er 'instructies1' en 'instructies2' van gemaakt.

Reply:
Ik bedoelde de rode tekst in de plaatjes zelf, niet de alt-tekst in Markdown.

Reply:
Maar weet of het nodig is dat aan te passen, dat stappenplan is toch tijdelijk.

Reply:
Ik heb net 2 nieuwe screenshots gemaakt en voorzien van Nederlandse tekst, die staan er nu in :)